csharp/dotnet/security/audit/missing-or-broken-authorization.yaml

```
@@ -15,6 +15,7 @@ rules:
     - 'CWE-862: Missing Authorization'
     cwe2021-top25: true
     cwe2022-top25: true
+    cwe2023-top35: true
```

```suggestion
    cwe2023-top25: true
```

Looks good @johanlindfors, just a a small typo 😄